packages/web-react/src/components/Modal/stories/ModalDialog.stories.tsx

```diff
@@ -9,6 +9,13 @@ const meta: Meta<typeof ModalDialog> = {
   title: 'Components/Modal',
   component: ModalDialog,
   argTypes: {
+    isDockedOnMobile: {
```

Comment:
Can you set defaults here as well? Otherwise storybook makes you click one more time.

Reply:
I thinks it's alright:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/74cbcbfb-a8b6-4f5c-8aaa-22266f0bdd58)